runtime/lib/Handlers/function/index.ts

```diff
@@ -83,6 +135,31 @@ export const FunctionHandler: HandlerFactory<FunctionCompiledNode, typeof utilsO
     const resolvedDefinition = resolveFunctionDefinition(definition, runtime.version!);
 
     try {
+      const parsedTransfers = NextBranchesDTO.safeParse(
+        runtime.variables.get(VoiceflowConstants.BuiltInVariable.FUNCTION_CONDITIONAL_TRANSFERS)
+      );
+
+      /**
+       * Case 1 - If there is a `parsedTransfers`, then we are resuming Function step execution after
+       *          obtaining user input
+       */
+      if (parsedTransfers.success) {
```

Comment:
should we handle this part only during 'request' stage?
```suggestion
      if (runtime.getAction() === Action.REQUEST && parsedTransfers.success) {
```

Reply:
Not sure, what is the `Action` enum used for?

Reply:
yeah you right